tuscan/tuscan_postprocess.py

```diff
@@ -179,5 +287,19 @@ def do_postprocess(args):
             pool.terminate()
             pool.join()
             exit(1)
-    pool.close()
-    pool.join()
+        pool.close()
```

Comment:
Yes, note that further up I moved the declaration of `pool` into the loop. A new pool is now created for each toolchain, rather than one pool for all toolchains. This is because the propagate_blockers function needs to run serially on each toolchain, so I wait for that function to finish before doing the next toolchain in parallel.